### DISCUSSION TITLE
Which version of Mongo is supported?

### DISCUSSION THREAD

**Discussion by razoreater** [Informatievlaanderen/VSDS-LDESServer4J#963] [Q&A] [answered]
At this moment we are testing with a version 7.0.1 MongoDB container.
But it seems that the VSDS package keeps failing with this version.

**Comment by Tomvbe** [comment, marked as answer]
We support the latest Mongo images. The server should work with `mongo:latest` which runs version 7.0.2.
If you are having database issues, there might be a bug or config issue, then it's best to create a github issue with more information.

**Comment by rorlic** [comment]
If you want to run the LDES server on a Raspberry PI4 with Raspbian64 (yes you can!) you can use [MongoDB 4.4.18](https://hub.docker.com/layers/library/mongo/4.4.18/images/sha256-766daf21cc3bdcd2f9cf0265473ce1d81c3712e4e625fd1ea72e626ed1378d72?context=explore) as the latest supported version due to MongoDB as of 4.4.19 requiring a minimum instruction set for the ARM64 which is not available on the PI4 processor.
